src/ocm/reducers/clusters/clustersSlice.ts

```diff
@@ -12,7 +12,7 @@ import { handleApiError, ocmClient } from '../../api';
 
 export const fetchClustersAsync = createAsyncThunk<Cluster[] | void>(
   'clusters/fetchClustersAsync',
-  async () => {
+  async (_, { rejectWithValue }) => {
```

Comment:
lets rename to something meaningful..ie `value`
```suggestion
  async (value, { rejectWithValue }) => {
```

Reply:
Done.